spec/mailers/communicart_mailer_spec.rb

```diff
@@ -209,16 +215,14 @@ def sender_names(mail)
   describe 'proposal_created_confirmation' do
     let(:mail) { CommunicartMailer.proposal_created_confirmation(proposal) }
 
-    it 'renders the subject' do
-      expect(mail.subject).to eq "Your request for #{proposal.public_identifier} has been sent successfully."
-    end
+    it_behaves_like "a Proposal email"
 
     it 'renders the receiver email' do
       expect(mail.to).to eq([proposal.requester.email_address])
     end
 
-    it 'renders the sender email' do
-      expect(mail.from).to eq(["reply@stub.gov"])
+    it "sets the sender name" do
+      expect(sender_names(mail)).to eq([nil])
```

Comment:
I'm not sure what this and the `it "sets the sender name"` above are actually testing. Seems like `it_behaves_like "a Proposal email"` covers the sender?